src/app/api/public/webhooks/clerk/route.ts

```
@@ -0,0 +1,72 @@
+import { WebhookEvent } from "@clerk/nextjs/server";
+import { headers } from "next/headers";
+import { NextResponse } from "next/server";
+import { Webhook } from "svix";
+
+export async function POST(req: Request) {
+  try {
+    const clerkWebhook = new Webhook(process.env.CLERK_SIGNING_SECRET!);
+    const body = await req.text();
+    const headersList = (await headers())
+      .entries()
+      .reduce((acc, [key, value]) => ({ ...acc, [key]: value }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)